tr.org.pardus.pen.yaml

```diff
@@ -0,0 +1,25 @@
+app-id: tr.org.pardus.pen
```

Comment:
How do I verify that you control this domain? https://pardus.org.tr/en/ does not list the GitHub orgnisation.

Reply:
Our github organisation link here:

![resim](https://github.com/user-attachments/assets/170ebb0b-d7b2-46db-94d0-81939d52ed4d)